homeassistant/components/huawei_lte/quality_scale.yaml

```diff
@@ -0,0 +1,76 @@
+rules:
+  # Bronze
+  action-setup: done
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
+  config-flow: done
+  dependency-transparency: done
+  docs-actions: done
+  docs-high-level-description: done
+  docs-installation-instructions: done
+  docs-removal-instructions: done
+  entity-event-setup: done
+  entity-unique-id: done
+  has-entity-name: done
+  runtime-data: todo
+  test-before-configure: done
+  test-before-setup: done
+  unique-config-entry: done
+
+  # Silver
+  action-exceptions: todo
```

Comment:
```py
    def press(self) -> None:
        """Press button."""
        if self.router.suspended:
            _LOGGER.debug(
                "%s: ignored, integration suspended", self.entity_description.key
            )
            return
```
We should not silently swallow exceptions for pressing a button (or well, this is not an exception, but we should rather raise an exception so the user sees why it didnt work)

Reply:
Noted in 2808787c845914a6f002b094c94337a280a98f03

Reply:
Status update in 5c75c33267f164bb72eb2ef80db21e03b0dbde54, improvements in https://github.com/home-assistant/core/pull/144200